### PR TITLE
*: update import path from coreos/gofail to etcd-io/gofail to fix ci

### PR DESCRIPTION
**Pull request by winkyao**
## What have you changed? (mandatory)

gofail change their path from `coreos/gofail` to `etcd-io/gofail` in this commit https://github.com/etcd-io/gofail/commit/be07ede6bb0f7f4944d17b1ef07933807d64d9f2, which will make tidb ci failed.

## What is the type of the changes? (mandatory)

- Improvement (non-breaking change which is an improvement to an existing feature)

## How has this PR been tested? (mandatory)

exist tests

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

no

## Does this PR affect tidb-ansible update? (mandatory)

no

## Does this PR need to be added to the release notes? (mandatory)

no

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
Should we cherry pick this to the release-2.0?

File: Gopkg.toml [hunk 1]
```diff
@@ -92,3 +88,7 @@ required = ["github.com/golang/protobuf/jsonpb"]
 [[constraint]]
   name = "github.com/opentracing/basictracer-go"
   version = "1.0.0"
+
+[[constraint]]
+  branch = "master"
```
Comment (zz-jason) [inline Gopkg.toml]: can we change `branch` to `version` to make tidb not depend on the `master` branch of `gofail`?

Comment (zz-jason) [reply Gopkg.toml]: In case they change the path again in the future.

Comment (winkyao) [reply Gopkg.toml]: gofail change the path, ci failed not caused by the version.

Comment (shenli) [reply Gopkg.toml]: Master is the only branch of go-fail. And they do not have any release.

**Pull request merged**